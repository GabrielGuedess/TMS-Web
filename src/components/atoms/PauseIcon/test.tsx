import { render } from '@testing-library/react';

import { PauseIcon } from '.';

describe('<PauseIcon />', () => {
  it('should render the PauseIcon correctly', () => {
    const { container } = render(<PauseIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
