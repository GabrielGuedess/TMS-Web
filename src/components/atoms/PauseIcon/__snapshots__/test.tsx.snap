// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PauseIcon /> should render the PauseIcon correctly 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 21C16.9706 21 21 16.9706 21 12C21 7.02944 16.9706 3 12 3C7.02944 3 3 7.02944 3 12C3 16.9706 7.02944 21 12 21Z"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-miterlimit="10"
    stroke-width="1.5"
  />
</svg>
`;
